integration-auth/src/main/resources/integration-auth-log-messages.properties

```diff
@@ -0,0 +1,12 @@
+auth.user.unregistered.message = Authentication invoked for unknown user - ID {0}
+auth.user.unregistered.solution = Unregistered user, check if the user exists
+auth.user.unauthorized.solution = Unauthorized user for reauthorization
+
+auth.session.token.unregistered.message = Authentication invoked for unknown user - ID
+auth.session.token.unregistered.solution = Unregistered token session on authentication context
+auth.session.token.unauthorized.message = Certificate authentication is unauthorized for the requested user - ID: {0}
```

Comment:
Certificate authentication is unauthorized for the requested user - Integration: {0}